client/src/modules/list/components/ListHeader.jsx

```diff
@@ -179,6 +190,17 @@ class ListHeader extends PureComponent {
     );
   };
 
+  handleTipVisibility = () => {
+    const { nameInputValue } = this.state;
+    const isItemNameEmpty = !_trim(nameInputValue);
+
+    if (isItemNameEmpty) {
+      this.setState({ isTipVisible: true });
+    }
+  };
+
+  hideTipAfterTimeout = () => this.setState({ isTipVisible: false });
```

Comment:
no timeout here,
change name

Reply:
the hint hides when the user enters something